### ISSUE TITLE
On click a particular user card, show its profile

### ISSUE THREAD

**Issue by Tanmoy741127** [CircuitVerse/mobile-app#264]
**Is your feature request related to a problem? Please describe.**
When you visit the members list of a group, you will get list like this.
<img src="https://user-images.githubusercontent.com/57363826/221542752-39d16d9e-e9ab-44a3-af12-80a16e8f9402.jpeg" width="250px"/>

The user card is not clickable.

**Describe the solution you'd like**
On clicking the card, it should show the Profile page of respective user.

**Additional context**
In the app, already the Profile Page has been pre-built. So you can try to use the same page and pass the user detals to that page to show Profile Page.

If you want API documentation of backend, you can explore that here : https://api.circuitverse.org/

**Are you working on this? (Yes/No)**
No

**Comment by sainideepanshu199** [comment]
can i work on this issue

**Comment by GauravDhak** [comment]
Hi , I would like to work on this issue. Can you assign this issue to me?

**Comment by lalitkumawat1m** [comment]
hey @tanmoysrt , I am willing to work on this issue, please assign me this issue.

**Comment by tanmoysrt** [comment]
> hey @tanmoysrt , I am willing to work on this issue, please assign me this issue.

cc @aman-singh7 @manjotsidhu

**Comment by harshsharma-11** [comment]
@aman-singh7 @manjotsidhu @GauravDhak  i want to work on this please assign it to me

**Comment by yash1378** [comment]
@tanmoysrt can i start working on this as there is no assignees?

**Comment by tanmoysrt** [comment]
> @tanmoysrt can i start working on this as there is no assignees?
> 

Sure , go ahead 
For discussion regarding ux, do conversation in slack #ux channel

**Comment by tanmoysrt** [comment]
@aman-singh7 @vedant-jain03 can you assign the issue to @yash1378 

> > @tanmoysrt can i start working on this as there is no assignees?
> 
> Sure , go ahead For discussion regarding ux, do conversation in slack #ux channel

**Comment by yash1378** [comment]
> @aman-singh7 @vedant-jain03 can you assign the issue to @yash1378
> 
> > > @tanmoysrt can i start working on this as there is no assignees?
> > 
> > 
> > Sure , go ahead For discussion regarding ux, do conversation in slack #ux channel

tha

> @aman-singh7 @vedant-jain03 can you assign the issue to @yash1378
> 
> > > @tanmoysrt can i start working on this as there is no assignees?
> > 
> > 
> > Sure , go ahead For discussion regarding ux, do conversation in slack #ux channel

thanx , if i need any help then how can i ask ? on the slack channel itself?

**Comment by tanmoysrt** [comment]
> > @aman-singh7 @vedant-jain03 can you assign the issue to @yash1378
> > 
> > > > @tanmoysrt can i start working on this as there is no assignees?
> > > 
> > > 
> > > Sure , go ahead For discussion regarding ux, do conversation in slack #ux channel
> 
> tha
> 
> > @aman-singh7 @vedant-jain03 can you assign the issue to @yash1378
> > 
> > > > @tanmoysrt can i start working on this as there is no assignees?
> > > 
> > > 
> > > Sure , go ahead For discussion regarding ux, do conversation in slack #ux channel
> 
> thanx , if i need any help then how can i ask ? on the slack channel itself?

Yes join the slack by this link https://join.slack.com/t/circuitverse-team/shared_invite/zt-p6bgler9-~8vWvsKmL9lZeYg4pP9hwQ
